sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -318,6 +318,11 @@ public FrameLayout getLayout() {
     return _layout.getPlayerLayout();
   }
 
+  @Override
```

Comment:
Could you add several words about this method? That this method serves for ima Measurement.

Reply:
I added additional info to Java doc into `public interface LayoutController`